src/WorkspaceDependencyViewExtension/WorkspaceDependencyViewExtension.cs

```diff
@@ -187,10 +187,6 @@ internal void DependencyRegen(WorkspaceModel ws, bool forceCompute = false)
                 HasDependencyIssue = string.IsNullOrEmpty(info.Path);
             }
 
-            var pythonPackageDependencies = ws.OnRequestPackageDependencies();
```

Comment:
Removed this as the OnRequestPackageDependencies was just setting the ironpython package details with a fixed version if the python node has a ironpython2 engine. Now, we will compute the dependency package directly when the engine is changed and the script is saved. Should handle Cpython and pythonnet3 cases as well.